peakina/readers/xml.py

```diff
@@ -0,0 +1,32 @@
+"""
+Module to add xml support
+"""
+from typing import Union
+
+import pandas as pd
+import xmltodict
+from jq import jq
+
+
+def transform_with_jq(data: Union[dict, list], jq_filter: str) -> list:
```

Comment:
Can you also had a jq filter for json files ?

Reply:
done :+1: